device-types/Fortinet/FAP-233G.yaml

```diff
@@ -0,0 +1,25 @@
+---
+manufacturer: Fortinet
+model: FortiAP 233G
+slug: fortinet-fap-233g
+part_number: FAP-233G
+u_height: 0.0
+is_full_depth: false
+airflow: passive
+weight: 0.8
+weight_unit: kg
+comments: '[FortiAP Series Datasheet](https://www.fortinet.com/content/dam/fortinet/assets/data-sheets/pdf/fortiap-series.pdf), [FortiAP 231G/233G QuickStart
+  Guide](https://docs.fortinet.com/document/fortiap/hardware/fortiap-231g-233g-qsg)'
+interfaces:
+  - name: lan-1-poe
+    type: 5gbase-t
```

Comment:
Thank you for your contribution,

Looking at the spec sheet provided this is a 2.5G port?